test-files/cmd/glacial/DiffConflatePartialLinearMatchTest.sh.stdout

```diff
@@ -0,0 +1,5 @@
+16c16
+< Total failing Hootenanny validators: 0
+---
+> Total failing Hootenanny validators: 0
+\ No newline at end of file
```

Comment:
I noticed that the `validation-report` files above all have the following indication:
![image](https://user-images.githubusercontent.com/13385275/135311783-0ad4d4c7-ad59-44e7-9862-d4b889f95256.png)
They don't have a newline at the end of the file.  GitHub doesn't like that and apparently this test output doesn't like it either.  I suggest ending the validation report files with a new line.

Reply:
thanks. Thought I had fixed that but think I just screwed it up again with a change from this morning.